packages/Uploader/src/components/DropZone/DropZone.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import tokens from "@paprika/tokens";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import UploadIcon from "@paprika/icon/lib/Upload";
+import { UploaderContext } from "../../Uploader";
+import * as div from "./DropZone.styles";
+
+export default function DropZone() {
+  const I18n = useI18n();
+  const uc = React.useContext(UploaderContext);
+  const isDraggingOver = uc.isDraggingOver;
+  const uploadIconColor = isDraggingOver ? tokens.color.purpleDarken10 : tokens.color.blackLighten50;
+
+  /* eslint-disable jsx-a11y/anchor-is-valid */
+  const body = isDraggingOver ? (
+    I18n.t("uploader.drop_files")
+  ) : (
+    <React.Fragment>
+      {I18n.t("uploader.drop_files_here_or")}&nbsp;
+      <a
+        href="#"
+        onClick={() => {
+          uc.refInput.current.click();
```

Comment:
The `href` is causing navigation without an `e.preventDefault()` in the `onClick` handler.  Also prevented by using a `<Button>`.

Reply:
Haha, thanks. I was stuck on this for too long and preventDefault escaped my mind.